admin_terraform_version_integration_test.go

```diff
@@ -100,25 +100,28 @@ func TestAdminTerraformVersions_CreateDelete(t *testing.T) {
 
 	client := testClient(t)
 	ctx := context.Background()
-	version := genSafeRandomTerraformVersion()
 
 	t.Run("with valid options", func(t *testing.T) {
-		sha := String(genSha(t))
 		opts := AdminTerraformVersionCreateOptions{
-			Version:          String(version),
-			URL:              String("https://www.hashicorp.com"),
-			Sha:              sha,
```

Comment:
Could we split this into two different tests and have one test with the URL/sha and no archs? It would be nice to confirm that the previous handling still works as intended for now!